projects/addon-mobile/directives/sidebar/sidebar.module.ts

```diff
@@ -1,13 +1,10 @@
 import {NgModule} from '@angular/core';
-import {TuiActiveZoneDirective} from '@taiga-ui/cdk';
-import {PolymorpheusModule} from '@tinkoff/ng-polymorpheus';
 
 import {TuiSidebarComponent} from './sidebar.component';
 import {TuiSidebarDirective} from './sidebar.directive';
 
 @NgModule({
-    imports: [TuiActiveZoneDirective, PolymorpheusModule],
-    declarations: [TuiSidebarDirective, TuiSidebarComponent],
+    imports: [TuiSidebarDirective, TuiSidebarComponent],
     exports: [TuiSidebarDirective],
 })
-export class TuiSidebarModule {}
+export class TuiSidebar {}
```

Comment:
There's no need for module at all, component is created dynamically, no need to import it.